docs/content/get-started/set-up.md

```diff
@@ -186,22 +192,20 @@ From the root of the `gdk-for-unity-fps-starter-project` repository:
 
 <%(#Expandable title="Manually clone the GDK for Unity")%>
 
-1. Clone the [GDK for Unity](https://github.com/spatialos/gdk-for-unity) repository alongside the FPS Starter Project so that they sit side-by-side:
-
+1\.  Clone the [GDK for Unity](https://github.com/spatialos/gdk-for-unity) repository alongside the FPS Starter Project so that they sit side-by-side:
 |     |     |
-| --- | --- |
 | HTTPS | `git clone https://github.com/spatialos/gdk-for-unity.git` |
 | SSH | `git clone git@github.com:spatialos/gdk-for-unity.git` |
 
   > The two repositories should share a common parent, like the example below:
 
-  ```text
+```
   <common_parent_directory>
       ├── gdk-for-unity-fps-starter-project
       ├── gdk-for-unity
-  ```
+```
 
-2. Navigate to the `gdk-for-unity` directory and checkout the pinned version which you can find in the `gdk.pinned` file, in the root of the `gdk-for-unity-fps-starter-project` directory.
+2\.  Navigate to the `gdk-for-unity` directory and checkout the pinned version which you can find in the `gdk.pinned` file, in the root of the `gdk-for-unity-fps-starter-project` directory.
```

Comment:
out of curiosity: what's that backslash for?

Reply:
I was having trouble getting this list to format correctly, it kept creating a new <ol> element for each numbered point so we'd have two 1. steps.  If you have a look at the live docs, it currently has step 1. twice. 

The slash just stops it from creating an <ol> and keeps it as a paragraph.